style/style.css

```diff
@@ -15,6 +15,7 @@ margin-left:-35px;
 
 html{
     scroll-behavior: smooth;
+    overflow-x: hidden;
```

Comment:
Are you encountering a horizontal scroll on the site? I think it is better we get to the root of the issue before directly just hiding the overflow.